previewers/betatest/js/video.js

```diff
@@ -9,5 +9,104 @@ function translateBaseHtmlPage() {
 
 function writeContent(fileUrl, file, title, authors) {
     addStandardPreviewHeader(file, title, authors);
-    $(".preview").append($("<video/>").prop("controls",true).append($('<source/>').attr("src",fileUrl)));
+
+    const queryParams = new URLSearchParams(window.location.search.substring(1));
+    const id = queryParams.get("datasetid");
+    const siteUrl = queryParams.get("siteUrl");
+    const versionUrl = `${siteUrl}/api/datasets/${id}/versions/`
```

Comment:
Someday it would make sense to have the retriever.js, which already gets this URL, either pass the json it gets or at least pass the signed URL (and other signed URLs, api key, etc.) through the writeContent method interface). Happy to avoid that now, but, per the comment below, it looks like at least the api key would have to be made available if this is to work with draft/restricted files.